web/src/app/review/page.tsx

```diff
@@ -100,9 +100,15 @@ export default function Review() {
     }, [activeCardSides.length, sidesVisible]);
 
     function daysSince(date: Date): number {
-        const now = new Date();
-        const diffMs = now.getTime() - date.getTime();
-        return Math.floor(diffMs / (1000 * 60 * 60 * 24));
+        const today = new Date();
+
+        today.setHours(0, 0, 0, 0);
+        date.setHours(0, 0, 0, 0);
```

Comment:
It is a bit crazy that there isn't a way to do solid date comparison without this tbh